src/sos/workflow_executor.py

```diff
@@ -1918,7 +1917,7 @@ def run_as_nested(self,
                             f'{len(sections)} pending step{"s" if len(sections) > 1 else ""}: {", ".join(sections)}'
                         ),
                     )
-                    raise exec_error
+                    raise e
```

Comment:
Please revert. `exec_error` was defined in the function and should not be replaced by KeybordInterruption.